test/app/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    nuodbaas = {
+      source  = "registry.terraform.io/nuodb/nuodbaas"
+      version = "0.2.0"
+    }
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "3.0.2"
+    }
+  }
+}
+
+provider "docker" { }
+
+provider "nuodbaas" { }
```

Comment:
You can accelerate the running time of the test by using the configuration:

```hcl
provider "nuodbaas" {
  timeouts = {
    projects = {
      create = "0"
    }
  }
}
```

Reply:
The difference is test running time is minimal (in either case, the database has to wait for the domain to be fully up and running before any of the processes start booting), but if we want to use this script as sample code, I want it to be simple instead of totally optimized.

Reply:
I changed it and (in local testing), it did not make a significant difference. Before, project took 1:00 to start and the database took 1:33, now project took 1-2s to create and the database took 2:17-2:30 to start.